services/gateway/sources/version2/mappings/newsfeed.js

```diff
@@ -19,8 +19,8 @@ module.exports = {
 	imageUrl: '=,string',
 	source: '=,string',
 	source_id: '=,string',
-	ctime: '=,number',
-	mtime: '=,number',
+	createdAt: '=,number',
```

Comment:
Does `v2` also need to be backward compatible?

Reply:
No, I had already discussed with @MichalTuleja and there is no need for it.